packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -103,7 +103,7 @@ export default function PostLockedModal() {
 					isTakeover: true,
 					user: {
 						name: received.lock_error.name,
-						avatar: received.lock_error.avatar_src,
+						avatar: received.lock_error.avatar_src_2x,
```

Comment:
Just out of curiosity, why did this value need to change to indicate 2x?

Reply:
This is how currently core provides 128px avatar when heartbeat updates the data - https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/misc.php#L1133.

Reply:
Thank you!